framework/.eslintrc.json

```diff
@@ -1,12 +1,12 @@
 {
 	"extends": ["../.eslintrc.json"],
 	"rules": {
-		"camelcase": "off",
+		"strict": "off",
```

Comment:
Commit's headline is misleading again. Why do you suddenly modify rule levels here?

Reply:
some rule levels are `warn` because it's not possible to change in one PR, but we don't want to keep the rule. Therefore, put it to warn and we can refactor when we change the code